HelpMyStreet.Utils/HelpMyStreet.Utils/Models/JobSummary.cs

```diff
@@ -24,6 +24,8 @@ public class JobSummary
         public bool ConsentForContact { get; set; }
         public DateTime DateRequested { get; set; }
         public RequestorType RequestorType { get; set; }
+        public bool? Archive { get; set; }
```

Comment:
It'd make the front end code simpler if this could be just bool.  Would that be possible?